deps/internal/cmds/get/topology.go

```diff
@@ -191,6 +334,7 @@ func topologyCommand(
 	addDependencyRequestFlags(cmd, req)
 
 	cmd.Flags().BoolVar(&tiered, "tiered", tiered, "Produce a tiered output instead of a flat stream")
+	cmd.Flags().StringVarP(&destinationModuleName, "destinationModuleName", "", "dest", "Get all the ways the module provided with --name flag is connected to this destination module")
```

Comment:
We could probably change the name/shorthand/message of the flag to express it better